index.bs

```diff
@@ -1534,6 +1535,17 @@ terminates the operation and returns an error.
 
 If the user refuses consent, the authenticator returns an appropriate error status to the client.
 
+When this method is invoked on a credential with ScopedCredentialType "ScopedCred_FIDO_UAF", the authenticator must perform the procedure as specified in [[FIDO-UAF-Authenticator-Commands]] section 6.3.  
```

Comment:
[Section 6.3](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html#sign-command) specifies parameter and response formats, which don't seem to be used here. Could you be clearer about how the inputs get from here into the [6.3.4](https://fidoalliance.org/specs/fido-uaf-v1.1-id-20170202/fido-uaf-authnr-cmds-v1.1-id-20170202.html#command-description-2) algorithm?